x-pack/solutions/observability/plugins/observability/public/components/alert_overview/helpers/map_rules_params_with_flyout.ts

```diff
@@ -77,6 +80,69 @@ const getPctAboveThreshold = (
   });
 };
 
+// Helper function to convert threshold values (divide by 100 for percent, by 8 for bits)
+const convertThresholdValues = (
+  threshold: number[],
+  metricField: string,
+  metric: string
+): number[] => {
+  return threshold.map((thresholdToFormat) => {
+    if (
+      metricField.endsWith('.pct') ||
+      (METRIC_FORMATTERS[metric] && METRIC_FORMATTERS[metric].formatter === 'percent')
+    ) {
+      thresholdToFormat = thresholdToFormat / 100;
+    } else if (
+      metricField.endsWith('.bytes') ||
+      (METRIC_FORMATTERS[metric] && METRIC_FORMATTERS[metric].formatter === 'bits')
+    ) {
+      thresholdToFormat = thresholdToFormat / 8;
+    }
+    return thresholdToFormat;
+  });
+};
```

Comment:
I think this way would be more readable, but completely up to you :) 

```suggestion
  return threshold.map((thresholdToFormat) => {
    if (
      metricField.endsWith('.pct') ||
      METRIC_FORMATTERS[metric]?.formatter === 'percent'
    ) {
      return thresholdToFormat / 100;
    } 
    
    if (
      metricField.endsWith('.bytes') ||
      METRIC_FORMATTERS[metric]?.formatter === 'bits'
    ) {
      return thresholdToFormat / 8;
    }
    
    return thresholdToFormat;
  });
};
```

Reply:
I tried to keep the same logic, just extracted that to a function, but either way works for me